data/niwei822.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Niwei Li",
+  "bio": "A former accountant turned software engineer interested in full stack web development."
```

Add a comma (**,**)at the end of line 3
```suggestion
  "bio": "A former accountant turned software engineer interested in full stack web development.",
```